s3-eventbridge-wildcard-sam/README.md

```diff
@@ -0,0 +1,113 @@
+# S3 to EventBridge to SQS with wildcard pattern match
+
+Publish events directly from Amazon S3 to EventBridge. There are multiple dynamic subfolders within the S3 buckets. EventBridge rule uses wildcard filters to select the required events from all the events and sends message to an Amazon SQS Queue.
```

Comment:
Please reword it in a way that explains what the pattern creates and what it does.

Reply:
Updated.